codegen/lib/src/field_validator_factory.dart

```diff
@@ -304,3 +317,7 @@ typedef Expression LazyViolation(Expression fieldValue);
 ///  - `false` if the constraint obeyed.
 ///
 typedef Expression LazyCondition(Expression fieldValue);
+
+/// A function of a field value expresion to code which prepares context before the constraint
```

Comment:
Please move the phrase starting with `before` to the next line. This won't leave a "check" alone in the dark.